sdk/test-utils/testing-recorder-new/package.json

```diff
@@ -100,7 +100,7 @@
     "typescript": "~4.2.0",
     "xhr-mock": "^2.4.1",
     "uuid": "^8.3.0",
-    "@azure/data-tables": "^13.0.2",
+    "@azure/data-tables": "^13.0.1",
```

Comment:
I am confused. In the recorder the version changes from 13.0.2 -> 13.0.1 and perf tests 13.0.2 -> 13.1.0

Reply:
Fixed, thank you!